python/subunit/filter_scripts/subunit2disk.py

```diff
@@ -19,5 +19,9 @@
 from subunit._to_disk import to_disk
 
 
-if __name__ == '__main__':
+def main():
     exit(to_disk())
```

Comment:
Shouldn't this be `sys.exit(...)`?

Reply:
> The site module (which is imported automatically during startup, except if the -S command-line option is given) adds several constants to the built-in namespace. ***They are useful for the interactive interpreter shell and should not be used in programs.***

* https://docs.python.org/3/library/constants.html?highlight=exit#constants-added-by-the-site-module